packages/react-components/src/components/Notifications.tsx

```diff
@@ -28,47 +34,32 @@ export interface NotificationsProps {
    * @defaultValue 2
    */
   maxNotificationsToShow?: number;
-}
 
-/**
- * All strings that may be shown on the UI in the {@link Notifications}.
- *
- * @beta
- */
-export interface NotificationsStrings {
   /**
-   * Unable to reach Chat service.
-   *
-   * This can mean:
-   *   - Incorrect Azure Communication Services endpoint was provided.
-   *   - User's network connection is down.
+   * Callback called when the dismiss button is triggered.
+   * Use this to control notifications shown when they dismissed by the user.
+   * Note this onDismiss function will affect all notifications in the same stack
    */
-  unableToReachChatService: NotificationBarStrings;
+  onDismissNotification?: (dismissedNotifications: ActiveNotification) => void;
 
   /**
-   * User does not have access to the Chat service.
-   * This usually means that either the Azure Communication Services endpiont or the token provided are incorrect.
-   */
-  accessDenied: NotificationBarStrings;
-
-  /**
-   * User is no longer on the thread.
+   * If set, notifications with {@link ActiveNotification.timestamp} older than the time this component is mounted
+   * are not shown.
    *
-   * See also: {@link NotificationsStrings.sendMessageNotInChatThread} for a more specific error.
-   */
-  userNotInChatThread: NotificationBarStrings;
-
-  /**
-   * Sending message failed because user is no longer on the thread.
-   */
-  sendMessageNotInChatThread: NotificationBarStrings;
-
-  /**
-   * A generic message when sending message fails.
-   * Prefer more specific error strings when possible.
+   * This is useful when using the {@link Notifications} with a stateful client that handles more than one call
+   * or chat thread. Set this prop to ignore notifications from previous call or chat.
+   *
+   * @defaultValue false
    */
-  sendMessageGeneric: NotificationBarStrings;
+  ignorePremountNotifications?: boolean;
```

Comment:
Is this the name we want to go with for this prop? Seems a bit confusing.
How about we do the opposite and name it:
`showOnlyCurrentNotifications`
`showOnlyCurrentCallNotifications`
`showOnlyCurrentCallAndChatNotifications`

Maybe a better name suggestion from someone else? Unless we agree 'premount' is easier to understand

Reply:
It's the same name we used for errorbar, might be more intuitive for users to learn when error bar is replaced since it's the same name

Reply:
This name also makes sense so that we are ignoring any of the errors that might come in before the composite or component mounts to the DOM as those errors might not matter (like muted)